src/router.js

```diff
@@ -21,7 +22,7 @@ const router = new Router({
       props: true,
     },
     {
-      path: '/org',
+      path: '/org/:userId?',
```

Comment:
What are your thoughts on making this path `org-chart` instead of `org` so that it matches page's name?

Reply:
We can change this again at some point. I moved it from `orgchart` to `org` since it overlapped with the orgchart service. But since this changes links to use the name `'Orgchart'` internally it's easy to change this once we settled on a schema.

Reply:
👍